published_versions.json

```
@@ -198,6 +198,200 @@
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch torchvision torchaudio"
+          }
+        },
+        "conda": {
+          "cuda10.2": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision torchaudio -c pytorch"
+          },
+          "cuda11.x": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision torchaudio -c pytorch"
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": null,
+            "command": "conda install pytorch torchvision torchaudio -c pytorch"
+          }
+        },
+        "libtorch": {
+          "accnone": {
+            "note": null,
+            "versions": {
+              "Download here:": "https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.11.0.zip"
+            }
+          },
+          "cuda10.2": {
+            "note": null,
+            "versions": {
+              "MacOS binaries do not support CUDA. Download CPU libtorch here:": "https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.11.0.zip"
+            }
+          },
+          "cuda11.x": {
+            "note": null,
+            "versions": {
+              "MacOS binaries do not support CUDA. Download CPU libtorch here:": "https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.11.0.zip"
+            }
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "versions": null
+          }
+        }
+      },
+      "windows": {
+        "pip": {
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "cuda10.2": {
+            "note": null,
+            "command": "pip3 install torch==1.11.0+cu102 torchvision==0.12.0+cu102 torchaudio===0.11.0+cu102 -f https://download.pytorch.org/whl/cu102/torch_stable.html"
+          },
+          "cuda11.x": {
+            "note": null,
+            "command": "pip3 install torch==1.11.0+cu113 torchvision==0.12.0+cu113 torchaudio===0.11.0+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html"
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on Windows",
+            "command": null
+          }
+        },
+        "conda": {
+          "cuda10.2": {
+            "note": null,
+            "command": "conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch"
```

Same as above, please either delete a command or point to 11.3
```suggestion
            "note": "CUDA-10.2 builds are not available on Windows",
            "command": "conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch"
```